patches/components-search_engines-template_url_prepopulate_data.cc.patch

```diff
@@ -0,0 +1,524 @@
+diff --git a/components/search_engines/template_url_prepopulate_data.cc b/components/search_engines/template_url_prepopulate_data.cc
```

Comment:
I think if in `brave/chromium_src/components/search_engines/template_url_prepopulate_data.cc` we move

`// Pull in definitions for Brave prepopulated engines. It's ugly but these need
// to be built as part of the search_engines static library.
#include "../../../components/search_engines/brave_prepopulated_engines.cc"
#include "../../../components/search_engines/brave_prepopulated_engines.h"`

up top after other includes then we won't need to patch this file at all (since we re-define `duckduckgo` and `qwant` in those 2 files).

Reply:
Took this suggestion and it works.